generators/docker-compose/index.js

```diff
@@ -78,37 +78,39 @@ module.exports = yeoman.Base.extend({
                 type: 'input',
                 name: 'directoryPath',
                 message: 'Enter the root directory where your gateway(s) and microservices are located',
-                default: '../',
-                validate: function (input) {
```

Comment:
yes I agree with you, about this disadvantage, but the yeoman mock bypass the validate. There are some topics about this
Maybe there is a better way...?